core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.ts

```diff
@@ -213,13 +209,11 @@ export class StoryEditorNavbarComponent implements OnInit {
 
   selectMainTab(): void {
     this.activeTab = this.EDITOR;
-    this.storyEditorNavigationService.navigateToStoryEditor();
```

Comment:
Why was this change made?

Reply:
Seems like a bug. Can you import `storyEditorNavigationService` in the constructor so it gets initialised? Ditto below.

Reply:
fixed this and tested on mobile view working fine now.

Reply:
Thanks for fixing this! :)